.github/workflows/reusable-build-and-push.yml

```diff
@@ -38,14 +38,12 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build
+      - name: Build and push
```

Comment:
If you change the name here we'll also need to change the name in the repo settings which blocks merges if this stage is not successfully run.

Reply:
it is not always pushing BTW, depends on the input

Reply:
I reverted it.